fastlane/Constants/Constants.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+class Constants
```

Comment:
Please don't create this file, you need to define the constants directly in the Dangerfile

Reply:
But how do we know the application's name before the user run make.sh command?

Reply:
Updated in ba738a1